### PR TITLE
Feature/baseline imports

### PR DESCRIPTION
**Pull request by suhaibMo**
changes addressed in submodule PR where parent is https://github.com/ebi-gene-expression-group/atlas-prod/pull/70

### CODE REVIEW

File: bin/import_baseline_proteomic_experiment.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```
Comment (pcm32) [inline bin/import_baseline_proteomic_experiment.sh]: Thanks for the refactoring Suhaib. This script in particular is the pipeline execution logic (the one that calls different modules). As such this should go in atlas-prod/exec directory.

Comment (pcm32) [reply bin/import_baseline_proteomic_experiment.sh]: (and not in the module).

File: bin/import_baseline_proteomic_experiment.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    		--include '*.png' \
+    		--include '*.bedGraph'\
+    		--include '*.Rdata' \
+    		--include '*.pdf' \
+    		--include '*.tsv.gz' \
+    		--exclude '*' \
+    		$@
+}
+
+copy_protein(){
```
Comment (pcm32) [inline bin/import_baseline_proteomic_experiment.sh]: please fix indentation once moved to atlas prod (this script).

Comment (pcm32) [reply bin/import_baseline_proteomic_experiment.sh]: (all through document)

File: bin/import_baseline_proteomic_experiment.sh [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+ 	
+ pushd $expTargetDir
+    # rename analysis methods
+    echo "renaming $expAcc"
+    rename_files $expAcc
+
+    # remove quotes and gename and make matrix in tsv format.
+    # remove genename column as we will decorate using recent Ensembl annotations.
+    echo "Remove genes names and make the matrix to tsv"
+    $projectRoot/bin/removeGeneNamesInDataMatrix.R ${expAcc}.tsv.undecorated.backup
```
Comment (pcm32) [inline bin/import_baseline_proteomic_experiment.sh]: assume that these are on the path (and make sure that the submodule that contains it has been added to the path).

Comment (pcm32) [reply bin/import_baseline_proteomic_experiment.sh]: Same with gxa_summarize_expression.pl

Comment (pcm32) [reply bin/import_baseline_proteomic_experiment.sh]: and with appendColnamesInDataMatrix.R

Comment (pcm32) [reply bin/import_baseline_proteomic_experiment.sh]: and with decorate_baseline_rnaseq_experiment.sh

**Review by pcm32 — COMMENTED** [review]
Great that we are having this as a module. Just leaving here some of the things we discussed during the day.

**Comment by suhaibMo** [comment]
@pcm32 So this import_baseline_proteomic_experiment.sh is removed from the submodule repo and moved to the main atlasprod/exec/import_baseline_proteomic_experiment.sh. So essentially this repo will be hosting the main import baseline script dependencies.

**Review by pcm32 — APPROVED** [review]
Great, thanks for refactoring as discussed.

**Comment by suhaibMo** [comment]
Thanks @pcm32

**Pull request merged**